ipaserver/install/ipa_restore.py

```diff
@@ -423,6 +423,9 @@ def run(self):
                 logger.info('Restarting SSSD')
                 sssd = services.service('sssd', api)
                 sssd.restart()
+                logger.info('Restarting oddjobd')
+                oddjobd = services.service('oddjobd', api)
+                oddjobd.restart()
```

Comment:
I'm afraid this won't be sufficient to just restart the service oddjobd. We need to make sure that it is also enabled (so that it starts after a reboot for instance), and there may also be some work related to sysrestore (to store if the service was running/enabled prior to the installation).

Reply:
Hello @flo-renaud :) Thank you, will change it.